KILTprotocol/kilt-workshop-101
# Feat+Fix: SDK version, diagram, improvements and style

## Fixes KILTProtocol/ticket#312

Description: Please see the ticket.

## How to test:

- Checkout this branch;
- `docsify serve .`;
- Go through the tutorial as a newcomer would, and check that everything works and makes sense;
- Check that the style, color, fonts are as in the screenshot below;
- **Please also click all the links to check they're correct**.

NOTE:
* The `Step 0: Setup` in the README won't work locally, that's expected.
* The ticket is also about reorganizing the folder structure; I'll do this after the first review.

<img width="1680" alt="Screenshot 2020-02-21 at 14 38 19" src="https://user-images.githubusercontent.com/9762897/75038935-d1925d00-54b7-11ea-960e-d7e3f3342e34.png">

## Checklist:
